cmd/grpcServer/main.go

```diff
@@ -31,7 +31,7 @@ const (
 
 func main() {
 	// create a tcp listener for your gRPC service
-	listener, err := net.Listen("tcp", "0.0.0.0:9090") //nolint:gosec // development use only
+	listener, err := net.Listen("tcp", "0.0.0.0:8088") //nolint:gosec // development use only
```

Comment:
Ahh I clashed with your centos management page.

Do you think we should make this an ENV VAR so we can change port at runtime?